pkg/cluster/kubernetes/resource/load_test.go

```diff
@@ -263,13 +264,57 @@ func debyte(r resource.Resource) resource.Resource {
 	return r
 }
 
+func TestSoftDecryption(t *testing.T){
+	enc := "encrypted data"
+	dec := `kind: Deployment
+metadata:
+	name: a-deployment
+`
+	strat := func(arg []byte) ([]byte, error){
+		return []byte(dec), nil
+	}
+	decrypted, err := softDecrypt([]byte(enc), strat)
+	if err != nil {
+		t.Error(err)
+	}
+	if string(decrypted) != dec {
+		t.Errorf("Expected: \n%#v\ngot:\n%#v", dec, decrypted)
+	}	
+}
+
+func TestSoftDecryptionNoMetadata(t *testing.T){
+	enc := "encrypted data"
+	dec := `kind: Deployment
+metadata:
+	name: a-deployment
+`
+	strat := func(arg []byte) ([]byte, error){
+		return nil, sops.MetadataNotFound
+	}
+	decrypted, err := softDecrypt([]byte(enc), strat)
+	if err != nil {
+		t.Error(err)
+	}
+	if string(decrypted) != enc {
+		t.Errorf("Expected: \n%#v\ngot:\n%#v", dec, decrypted)
```

Comment:
This error doesn't make sense (since `dec` wasn't returned). In fact, I would remove `enc` end only use `dec` (renaming it to `plaintextManifest`) or something in those lines.